src/components/editCqlLibrary/EditCqlLibrary.tsx

```diff
@@ -356,203 +358,199 @@ const EditCqlLibrary = () => {
         success={success}
         outboundAnnotations={outboundAnnotations}
       />
-      <div className="flow-container">
-        <div id="left-panel">
-          <div tw="flex-grow " data-testid="cql-library-editor-component">
-            {!activeSpinner && (
-              <>
-                <div id="header-editor-row">
-                  <IconButton
-                    data-testid="editor-search-button"
-                    aria-label="search button"
-                    style={{
-                      color: "#0073c8",
-                    }}
-                    onClick={toggleSearch}
-                  >
-                    <Search />
-                  </IconButton>
-                </div>
-                <CqlLibraryEditor
-                  value={formik.values.cql}
-                  onChange={onChange}
-                  readOnly={!formik.values.draft || !canEdit}
-                  valuesetSuccess={valuesetSuccess}
-                  valuesetMsg={valuesetMsg}
-                  inboundAnnotations={elmAnnotations}
-                  setOutboundAnnotations={setOutboundAnnotations}
-                />
-              </>
-            )}
-            {activeSpinner && (
-              <div
+      <div
+        className="allotment-wrapper"
+        data-testid="cql-library-editor-component"
+      >
+        <Allotment>
+          <Allotment.Pane>
+            <div id="left-panel" tw="flex-grow">
+              <IconButton
+                data-testid="editor-search-button"
+                aria-label="search button"
                 style={{
-                  display: "flex",
-                  justifyContent: "center",
-                  height: "calc(100vh - 135px)",
```

Comment:
Lets keep the old calc until we implement a programmatic check for making sure it's always the right size